.golangci.yaml

```diff
@@ -114,6 +126,9 @@ issues:
     - unused
     - revive
     - stylecheck
+    - noctx
```

Comment:
This should be sorted alphabetically.

Reply:
Its sorted already, no ?

Reply:
In which alphabet the order is u, r, s, n, u, g? :smile:

Reply:
Oh crap, my bad. I was looking into the enable list. 

```
enable:
  - asasalint
  - asciicheck
  - bidichk
  - bodyclose
  # - containedctx
  - dogsled
  - dupword
  - durationcheck
  - errcheck
  - errchkjson
  - exportloopref
  - gci
  - ginkgolinter
  - goconst
  - gocritic
  - godot
  - gofmt
  - goimports
  - goprintffuncname
  - gosec
  - gosimple
  - govet
  - importas
  - ineffassign
  - loggercheck
  - misspell
  - nakedret
  - nilerr
  - noctx
  - nolintlint
  - nosprintfhostport
  - prealloc
  - predeclared
  - revive
  - rowserrcheck
  - staticcheck
  - stylecheck
  - thelper
  - typecheck
  - unconvert
  - unparam
  - unused
  - usestdlibvars
  - whitespace
```

Reply:
fixed